readthedocs/core/resolver.py

```diff
@@ -1,13 +1,15 @@
 """URL resolver for documentation."""
 
 from __future__ import absolute_import
-from builtins import object
+
 import re
+from urllib.parse import urlunparse
 
+from builtins import object
```

Comment:
If we are merging this _after_ deprecating Py2, we should remove this and also the `__future__` one.

Reply:
I just run isort on the files, I'll manage the conflicts after python2 support is dropped.